docs/hazmat/primitives/asymmetric/rsa.rst

```diff
@@ -167,6 +167,25 @@ There is a shortcut to sign sufficiently short messages directly:
     ...     hashes.SHA256()
     ... )
 
+
+If you have pre-hashed your data you can sign using that as well:
```

Comment:
I feel like this shouldn't be a part of the primary narrative, since it's relatively obscure. What do you think?